test/built-ins/Array/prototype/toReversed/get-descending-order.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2021 Igalia. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.prototype.toReversed
+description: >
+  Array.prototype.toReversed gets the array elements from the last one to the first one.
+info: |
+  Array.prototype.toReversed ( )
+
+  ...
+  2. Let len be ? LengthOfArrayLike(O).
+  ...
+  5. Repeat, while k < len
+    a. Let from be ! ToString(𝔽(len - k - 1)).
+    ...
+    c. Let fromValue be ? Get(O, from).
+    ...
+features: [change-array-by-copy]
+includes: [compareArray.js]
+---*/
+
+var order = [];
+var arrayLike = {
+  length: 3,
+  get 0() {
+    order.push(0);
+  },
+  get 1() {
+    order.push(1);
+  },
+  get 2() {
+    order.push(2);
+  },
+};
+
+Array.prototype.toReversed.call(arrayLike);
```

Comment:
Can have this for `[].toReversed()` as well?

(And similar assertion to all other `get` order tests in this PR)

Reply:
Sorry, I'm not sure I understand your comment -- do you mean adding the same test but with length == 0 and verifying that toReversed() doesn't access any elements, like:

`var order = [];
var arrayLike = {
  length: 0,
  get 0() {
    order.push(0);
  },
  get 1() {
    order.push(1);
  },
  get 2() {
    order.push(2);
  },
};

Array.prototype.toReversed.call(arrayLike);

assert.compareArray(order, []);
`
Or something else? (I'm working with Nicolò to finish this PR.)

Reply:
I think I understand what you meant now -- take a look at toReversed/get-descending-order and toSpliced/elements-read-in-order and let me know if that's what you had in mind.